Source/DafnyLanguageServer/Program.cs

```diff
@@ -14,15 +16,17 @@ public class Program {
       var configuration = CreateConfiguration(args);
       InitializeLogger(configuration);
       try {
+        var cancelLanguageServer = new CancellationTokenSource();
```

Comment:
This `CancellationTokenSource` is now unnecessary. The call to `cancelLanguageServer.Cancel()` can be replaced with `server.ForcefulShutdown()`. This change would make my comment regarding the disposal of the registration obsolete too since it can be omitted.

Reply:
> Any reason for this? I see that the target buffer is a MemoryStream; thus, the actions tend to be synchronous anyway. However, it would make the warning suppression obsolete, as well as the use of Wait that would require manual unwrapping of the nested exceptions.

The reason is to make the return type a `string` instead of `Task<string>`. I don't see why computing a static value would require a `Task`. But.. I guess this goal is not worth the trouble I'm creating :-)